ChiragTagadiya/MyTwiLite
# Develop 1.0.0

# Description
> This PR to add the following features

# Features
- Sign Up, Sign In and Sign Out features
- User can post text or a single image, or both of them, to a timeline
- User can delete their own post
- Post/Timeline view page

# How to test?
- Create a user using Sign Up
- Check-out all posts on "Timelines" tab
- Tap on "+" icon to add a timeline
- Add description and/or picture for the timeline
- Tap on "Post" button to post it on the timeline
- Go to "My Timeline" to see and manage all timeline
- Tap on delete icon to delete a timeline
- Tap on "Profile" to see user profile information
- Tap on "Log Out" to do log out.

# Evidence
<video width=350, src="https://user-images.githubusercontent.com/46391124/215263028-c3c44ef0-3f22-4f1a-bccc-f9e22c9d3021.mov">
